doc/source/getting-started/installation.rst

```diff
@@ -66,6 +66,27 @@ To install this package with ``conda`` run::
 .. asciinema:: 507565
 
 
+Pixi
+~~~~
+
+`Pixi <https://pixi.sh/>`_ is a modern package management tool that provides fast, reliable, and reproducible software environments. It can be used as an alternative to conda/mamba for managing PyVista installations.
+
+To install PyVista using pixi, first ensure you have pixi installed (see `pixi installation instructions <https://pixi.sh/latest/#installation>`_), then run::
+
+    pixi add pyvista
+
+This will add PyVista to your ``pixi.toml`` file and install it in your project environment. To use PyVista in your pixi environment::
+
+    pixi shell
+    python -c 'import pyvista; print(pyvista.__version__)'
+
+For global installation, you can use::
+
+    pixi global install pyvista
```

Comment:
Not sure I'd recommend installing `pyvista` in the global workspace.

Isn't that more for packages with entry-points which allows you to easily use them from the CLI?

Reply:
True. Let's remove this!